pyproject.toml

```diff
@@ -60,7 +60,7 @@ python = ">=3.7.0,<4"
 django = ">=2"
 importlib_metadata = { version = ">=4", python = "~3.7" }
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
```

Comment:
this, and lockfile regeneration, are from using the latest Poetry - c2c77ba9d9fc6474b1f381dbfeb6fbe2a5b8aebe - commit can be cherry-picked and merged beforehand